pkg/auth/iguazio/v4/session.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2025 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v4
+
+import "github.com/nuclio/nuclio/pkg/auth/iguazio"
+
+type Session struct {
+	*iguazio.AbstractSession
+}
```

Comment:
Is this really needed?

Reply:
@TomerShor yes, just wrapping to make it v1.Session and v2.Session and avoid using abstract anywhere